lib/streamlit/elements/write.py

```diff
@@ -145,8 +145,8 @@ def write(self, *args, **kwargs):
         >>> st.write(c)
 
         ..  output::
-            https://static.streamlit.io/0.25.0-2JkNY/index.html?id=8jmmXR8iKoZGV4kXaKGYV5
-            height: 200px
+            https://share.streamlit.io/streamlit/docs/main/python/api-examples-source/charts.vega_lite_chart.py
```

Comment:
Same here, is this correct that vega_lite_chart is the correct app for st.write?

Reply:
Yup, the output of the code is visually identical to that of the Altair chart and Vega lite chart examples, so I reuse the same app.